hamocc/mo_cyano.F90

```diff
@@ -75,7 +75,7 @@ subroutine cyano(kpie,kpje,kpke,kbnd,pddpo,omask,ptho)
     do j=1,kpje
       do i=1,kpie
         if (omask(i,j) > 0.5) then
-          do k=1,merge(kwrbioz(i,j),kmle(i,j),leuphotic_cya) ! if leuphotic_cya=.true., do bluefix only in euphotic zone
+          do k=1,kwrbioz(i,j) ! if leuphotic_cya=.true., do bluefix only in euphotic zone
```

Comment:
```suggestion
          do k=1,kwrbioz(i,j) ! do bluefix only in euphotic zone
```
The comment on this line can also be changed (or removed)

Reply:
Recognized it, when you approved :-) - sorry for that. Done.